blackbox/docs/sql/queries.txt

```diff
@@ -569,7 +569,7 @@ contains an element 'netball'::
 It can also be used on array literals::
 
     cr> select name, race['interests'] from locations
-    ... where name = ANY (['Bartledan', 'Algol'])
+    ... where name = ANY(ARRAY['Bartledan', 'Algol'])
```

Comment:
if you'd keep the difference between array literals and constructor, this wouldn't be correct as it describes it as usage of array literals